### PR TITLE
Fix readme example

### LINKED ISSUE #24: Error when running example
In the sample code it states:
from adafruit_bmp280 import adafruit_bmp280
while it should state:
from adafruit_bmp280 import Adafruit_BMP280

### PR DESCRIPTION
**Pull request by FoamyGuy**
Fixes #24 
This change updates the readme example to use code from simpletest. Also Fixes a few pylint ssues in simpletest.

Tested on CLUE

### CODE REVIEW

File: examples/bmp280_simpletest.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 # import digitalio # For use with SPI
 import busio
-
 import adafruit_bmp280
 
 # Create library object using our Bus I2C port
-i2c = busio.I2C(board.SCL, board.SDA)
-bmp280 = adafruit_bmp280.Adafruit_BMP280_I2C(i2c)
+i2c = busio.I2C(board.SCL, board.SDA)  # pylint: disable=invalid-name
+bmp280 = adafruit_bmp280.Adafruit_BMP280_I2C(i2c)  # pylint: disable=invalid-name
```
Comment (tannewt) [inline examples/bmp280_simpletest.py]: Examples should be exempt from these checks: https://github.com/adafruit/Adafruit_CircuitPython_BMP280/blob/master/.github/workflows/build.yml#L52

Comment (FoamyGuy) [reply examples/bmp280_simpletest.py]: Thank you Scott
When I check pylint locally I use a command like this:
```
pylint --rcfile .pylintrc path\to\somefile.py
```
I don't have much experience with the github workflows like this forgive me if it's obvious, but I don't see any reference to the `.pylintrc` file around that section you linked to. 

Is the command like I've used here the correct way for me to check pylint locally? 

Is there a different `.pylintrc` for the examples? or do they skip pylinting entirely?

Comment (jerryneedell) [reply examples/bmp280_simpletest.py]: @FoamyGuy if you look at https://github.com/adafruit/Adafruit_CircuitPython_BMP280/blob/master/.github/workflows/build.yml#L52  you can see the pylon command used for the examples.

Comment (FoamyGuy) [reply examples/bmp280_simpletest.py]: Ah, I think I get it now, there are a few extra things disabled via the flag `--disable=missing-docstring,invalid-name,bad-whitespace` passed in when it's run. Thank you Jerry.

I pushed a change to remove them from the example script the other day. But I just noticed that I've left them in the readme. I'll push another commit to fix that shortly.

**Review by kattni — APPROVED** [review]
Thanks for the fix!

**Pull request merged**